interfaces/builtin/multipass.go

```diff
@@ -0,0 +1,50 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const multipassSummary = `allows access to the Multipass socket`
+
+const multipassBaseDeclarationSlots = `
+  multipass:
+    allow-installation: false
+    deny-connection: true
+    deny-auto-connection: true
+`
+
+const multipassConnectedPlugAppArmor = `
+# Description: allow access to the Multipass daemon socket.
```

Comment:
Can you adjust to include: `This gives privileged access to the system via the mulipass socket API`?

Reply:
This is gone...

Reply:
Gone or done?

Reply:
Gone. This was a multipass client interface, which became unnecessary after refactoring.